unlock-app/src/config.js

```diff
@@ -75,6 +75,9 @@ export default function configure(
   let services = {}
   let supportedProviders = []
   let blockTime = 8000 // in mseconds.
+  let chainExplorerUrlBuilders = {
+    etherScan: () => false,
```

Comment:
how will this be used in the app? to disable the button?

Reply:
Yeah, basically to have a do-nothing link. Open to other ideas on how to handle this